web/src/utils/columns.ts

```diff
@@ -148,63 +148,67 @@ export const getCommonColumns = (t: TFunction, withConcatenatedFields = true): C
       name: t('Names'),
       isSelected: false,
       value: f => getSrcOrDstValue(f.fields.SrcK8S_Name, f.fields.DstK8S_Name),
-      sort: (a, b, col) => compareStrings(col.value(a) as string, col.value(b) as string),
+      sort: (a, b, col) => compareStrings((col.value(a) as string[]).join(''), (col.value(b) as string[]).join('')),
       width: 15
     },
     {
       id: ColumnsId.type,
       name: t('Kinds'),
       isSelected: false,
       value: f => getSrcOrDstValue(f.fields.SrcK8S_Type, f.fields.DstK8S_Type),
-      sort: (a, b, col) => compareStrings(col.value(a) as string, col.value(b) as string),
+      sort: (a, b, col) => compareStrings((col.value(a) as string[]).join(''), (col.value(b) as string[]).join('')),
       width: 10
     },
     {
       id: ColumnsId.owner,
       name: t('Owners'),
       isSelected: false,
       value: f => getSrcOrDstValue(f.labels.SrcK8S_OwnerName, f.labels.DstK8S_OwnerName),
-      sort: (a, b, col) => compareStrings(col.value(a) as string, col.value(b) as string),
+      sort: (a, b, col) => compareStrings((col.value(a) as string[]).join(''), (col.value(b) as string[]).join('')),
       width: 15
     },
     {
       id: ColumnsId.ownertype,
       name: t('Owner Kinds'),
       isSelected: false,
       value: f => getSrcOrDstValue(f.fields.SrcK8S_OwnerType, f.fields.DstK8S_OwnerType),
-      sort: (a, b, col) => compareStrings(col.value(a) as string, col.value(b) as string),
+      sort: (a, b, col) => compareStrings((col.value(a) as string[]).join(''), (col.value(b) as string[]).join('')),
       width: 10
     },
     {
       id: ColumnsId.namespace,
       name: t('Namespaces'),
       isSelected: false,
       value: f => getSrcOrDstValue(f.labels.SrcK8S_Namespace, f.labels.DstK8S_Namespace),
-      sort: (a, b, col) => compareStrings(col.value(a) as string, col.value(b) as string),
+      sort: (a, b, col) => compareStrings((col.value(a) as string[]).join(''), (col.value(b) as string[]).join('')),
       width: 15
     },
     {
       id: ColumnsId.addr,
       name: t('IP'),
       isSelected: false,
       value: f => getSrcOrDstValue(f.fields.SrcAddr, f.fields.DstAddr),
-      sort: (a, b, col) => compareIPs(col.value(a) as string, col.value(b) as string),
+      sort: (a, b, col) => compareIPs((col.value(a) as string[]).join(''), (col.value(b) as string[]).join('')),
       width: 10
     },
     {
       id: ColumnsId.port,
       name: t('Ports'),
       isSelected: false,
       value: f => getSrcOrDstValue(f.fields.SrcPort, f.fields.DstPort),
-      sort: (a, b, col) => comparePorts(col.value(a) as number, col.value(b) as number),
+      sort: (a, b, col) =>
+        comparePorts(
+          (col.value(a) as number[]).reduce((a, b) => a + b, 0),
```

Comment:
hmm summing the ports is pretty weird :) but ok for a quick fix!
Maybe we should later do something more elaborated, like comparing source, then if equal, comparing dest.

Reply:
yes I agree. I was also thinking about having 4 sort states on these columns:
- Source ASC
- Source DESC
- Destination ASC
- Destination DESC

We'll see that after table / columns refactoring

Thanks